### DISCUSSION TITLE
[Bounty] $10 to apply crawling under tables from Cortical Borers to any mob, not just cortical borers

### DISCUSSION THREAD

**Discussion by ReturnToZender** [Bubberstation/Bubberstation#1804] [ideas/bounties]
May need some slight balancing work, in which case price is negotiable.

Currently, cortical borers are the only mob that can crawl under a table. I want to change that so my fellows can be under my table to, uh, hide from Security, yes.

**Comment by Odairu** [comment]
This is completed, waiting on merge upstream, then for it to make its way here

**Comment by ReturnToZender** [comment]
Merged upstream, waiting to make its way down
